pkg/sqlite/query.go

```diff
@@ -94,7 +94,7 @@ func (s *SQLQuerier) GetPackage(ctx context.Context, name string) (*registry.Pac
 		},
 	}
 
-	if rows.Next() {
+	for rows.Next() {
```

Comment:
nit: we can collapse the `rows.Scan` for the first item into the loop, prepare the `channels` array first and then populate the `&registry.PackageManifest{`.